lib/charms/observability_libs/v0/juju_topology.py

```diff
@@ -0,0 +1,301 @@
+# Copyright 2022 Canonical Ltd.
```

Comment:
Why do you need this lib? I don't see it imported anywhere but could have just missed it.

Reply:
Looks like a transitive dependency used by Loki and Prometheus libraries.

Reply:
Yeap, that is the reason.